### ISSUE TITLE
Override default values for user settings by administrator

### ISSUE THREAD

**Issue by cliqsocial** [misskey-dev/misskey#7069]
There has been many complaints about instance ticker flooding the feeds

This is a feature request so setting can be changed by admin

Right now you have to modify the code

Not sure why https://github.com/syuilo/misskey/issues/6983 was closed

**Comment by rinsuki** [comment]
> 各種設定値のインスタンスデフォルト値定義機能は計画してるけど実装はまだ先になりそう
> https://github.com/syuilo/misskey/issues/6983#issuecomment-757096105

**Comment by cliqsocial** [comment]
Yes if it is planned then issue should remain open. It's ok that's why this issue was created. Thank you @rinsuki
